### PR TITLE
HFP-2065 Fix HTML encoding on result screen

### PR DESCRIPTION
**Pull request by otacke**

### CODE REVIEW

File: js/flashcards.js [hunk 1]
```diff
@@ -478,7 +478,7 @@ H5P.Flashcards = (function ($, XapiGenerator) {
 
       $('<div/>', {
         'class': 'h5p-results-question',
-        'text': card.text
+        'text': C.$converter.html(card.text || '').text()
```
Comment (fnoks) [inline js/flashcards.js:481]: Shouldn't it be the following instead?
`html: card.text `

Comment (otacke) [reply js/flashcards.js:481]: I tried to keep it consistent with https://github.com/h5p/h5p-flashcards/blob/master/js/flashcards.js#L131

Comment (fnoks) [reply js/flashcards.js:481]: I guess that is done that way since it is comparing user input with the correct answer.

Comment (otacke) [reply js/flashcards.js:481]: Should I amend the pull request or would you like to change the line yourself?

Comment (otacke) [reply js/flashcards.js]: @fnoks Just changed it.

**Pull request merged**